lua/telescope/pickers/layout_strategies.lua

```diff
@@ -468,46 +488,54 @@ layout_strategies.cursor = make_documented_layout(
     local width_opt = layout_config.width
     local width = resolve.resolve_width(width_opt)(self, max_columns, max_lines)
 
-    local max_width = (width > max_columns and max_columns or width)
-
     local bs = get_border_size(self)
 
+    -- Cap over/undersized height
+    height = math.min(height, max_lines)
+    height = math.max(height, 2 + (3 * bs))
```

Comment:
```
height = bounded(height, max_lines, 2 + (3 *bs))

height = border_bounded(height, max_lines, 2)
-- height = math.min(height, max_lines)
--   height = math.max(height, offset + (3 * bs))
```

Reply:
Problem is the multiplier is different in different layouts

Reply:
But could factor into a function to make clearer what's going on